.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
Reconsidered this, and now I think we should include ``.pxi`` under git actually. Otherwise, Travis cache can't work because ``.pxi`` is always newly generated?